server/buttonAlerts.js

```diff
@@ -41,15 +41,15 @@ async function handleValidRequest(button, numButtonPresses) {
     const currentTime = await db.getCurrentTime(pgClient)
 
     if (currentSession === null || currentTime - currentSession.updatedAt >= helpers.getEnvVar('SESSION_RESET_TIMEOUT')) {
-      currentSession = await db.createSession(button.id, CHATBOT_STATE.STARTED, numButtonPresses, null, null, null, pgClient)
+      currentSession = await db.createSession(button.id, CHATBOT_STATE.STARTED, numberOfAlerts, null, null, null, pgClient)
     } else {
-      currentSession.incrementButtonPresses(numButtonPresses)
+      currentSession.incrementButtonPresses(numberOfAlerts)
       await db.saveSession(currentSession, pgClient)
     }
 
     await db.commitTransaction(pgClient)
 
-    if (currentSession.numButtonPresses === 1) {
+    if (currentSession.numberOfAlerts === 1) {
```

Comment:
All the rest of the ones in this file though are good as `numberOfAlerts` because they are all coming from the `currentSession` object that was retrieved from the DB.